### ISSUE TITLE
Matlab version

### ISSUE THREAD

**Issue by kvlane** [HMS-IDAC/SarcTrack2#1]
I'm running into a lot of errors that I think might have to do with the Matlab version I am using. With the original SarcTrack, it only works for me on Matlab R2018b, so I figured that might be similar here. I've tried 2018b through 2020b, none of which have worked for me. I often get the error "Unrecognized function or variable 'getIpOptions'" when running the dwProcessFolder.m script (after dwProcessFrame.m works fine).

Is this an error because I'm missing a toolbox or could it be because of the Matlab version I'm using?

**Comment by cicconet** [comment]
Matlab 2018b should work. I don't recognize that error. Can you send the full error message?

**Comment by kvlane** [comment]
Here's a screenshot of the error! 
![Screen Shot 2022-02-23 at 11 04 19 AM](https://user-images.githubusercontent.com/66803284/155416795-92294965-94b2-45de-8255-6b8616a6cfdb.png)

**Comment by cicconet** [comment]
Do you have all Matlab packages installed or just the base version?

**Comment by kvlane** [comment]
What do you mean by all Matlab packages? These are the add-ons I have (screenshot)
<img width="699" alt="Screen Shot 2022-02-28 at 9 51 53 AM" src="https://user-images.githubusercontent.com/66803284/156033217-d32553b3-6950-4c32-bb1b-bcd0fec26c6b.png">
These are what came up as needed when running the code.

**Comment by cicconet** [comment]
Yes that's what I meant. I think you need the Optimization toolbox. Or if you have it installed, try re-installing it. That's what was suggested here: https://www.reddit.com/r/matlab/comments/jp20ph/help_with_fmincon_error_message/

**Comment by kvlane** [comment]
Okay great, thank you! That solved that error, but now I'm getting this error. 
<img width="674" alt="Screen Shot 2022-03-01 at 10 34 38 AM" src="https://user-images.githubusercontent.com/66803284/156228707-1587d128-cae0-4626-a7d2-44af97b4e27c.png">

**Comment by kvlane** [comment]
Any updates/thoughts about the error I posted? I'm not sure how else to troubleshoot.
